configuration_test.go

```diff
@@ -38,6 +38,20 @@ func mustLoadConfiguration(path string) {
 	}
 }
 
+func removeFile(t *testing.T, filename string) {
+	err := os.Remove(filename)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func fileClose(file *os.File) {
```

Comment:
Where do you use this function, because I cannot find?

Reply:
Oh, I forgot to delete it. It was a first try to fix one of the ignored errors. Thank you for pointing out